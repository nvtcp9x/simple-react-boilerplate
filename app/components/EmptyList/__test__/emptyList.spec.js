import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

import { EmptyList } from '../index';

Enzyme.configure({ adapter: new Adapter() });

describe('EmptyList component', () => {
  const component = shallow(<EmptyList />);

  it('should ...', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
