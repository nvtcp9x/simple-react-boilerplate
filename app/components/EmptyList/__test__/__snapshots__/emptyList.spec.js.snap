// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyList component should ... 1`] = `
<Row
  tag="div"
>
  <h2>
    No watchList
  </h2>
</Row>
`;
